cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -226,3 +227,41 @@ message SegYSeismicResponse {
 message SearchFilesRequest {
     SearchSpec spec = 1;
 }
+
+message SearchJobStatusRequest {
+    message Filter {
+        message Id {
+            oneof id {
+                string job_id = 1;
+                string file_id = 2;
```

Comment:
On a second thought, should we start calling these file_uuid?

Reply:
Can do